packages/core/src/client.ts

```diff
@@ -111,13 +111,13 @@ export interface ClientOptions {
    * This is the basis for how `urql` handles GraphQL operations, and exchanges handle the creation, execution,
    * and control flow of exchanges for the `Client`.
    *
-   * When this option is left out, the `Client` defaults to a list of {@link defaultExchanges}. By default, these
-   * will implement deduping, caching (via a document cache), and fetching (GraphQL over HTTP).
+   * To easily get started you should consider using the {@link dedupExchange}, {@link cacheExchange} and {@link fetchExchange}
+   * these are all exported from the core package.
    *
    * @see {@link https://formidable.com/open-source/urql/docs/architecture/#the-client-and-exchanges} for more information
    * on what `Exchange`s are and how they work.
    */
-  exchanges?: Exchange[];
+  exchanges: Exchange[];
```

Comment:
We could define this as `[Exchange, ...Exchange[]]` to enforce that at least one exchange should be passed. Further, we can then add an `@example` TSDoc here, which explains the default for new-comers ✨

Reply:
Ha, I just realised that that'd mean updating a gazillion tests, so maybe not 🤷